### PR TITLE
Indicator api docs

### PR DESCRIPTION
**Pull request by glind**
## Purpose
Update indicator api docs and cleanup

## Approach
used help_text attribute and removed unused fields

### CODE REVIEW

File: workflow/models.py [hunk 1]
```diff
@@ -1097,54 +1097,51 @@ class WorkflowLevel2(models.Model):
...
-    level2_uuid = models.CharField(max_length=255, verbose_name='WorkflowLevel2 UUID', default=uuid.uuid4, unique=True, blank=True)
-    short = models.BooleanField(default=True, verbose_name="Short Form (recommended)")
-    workflowlevel1 = models.ForeignKey(WorkflowLevel1, verbose_name="Program", related_name="workflowlevel2")
+    level2_uuid = models.CharField(max_length=255, verbose_name='WorkflowLevel2 UUID', default=uuid.uuid4, unique=True, blank=True, help_text="Unique ID")
+    workflowlevel1 = models.ForeignKey(WorkflowLevel1, verbose_name="Program", related_name="workflowlevel2", help_text="Primary Workflow")
     parent_workflowlevel2 = models.IntegerField("Parent", default=0, blank=True)
-    milestone = models.ForeignKey("Milestone", null=True, blank=True, on_delete=models.SET_NULL)
-    date_of_request = models.DateTimeField("Date of Request", blank=True, null=True)
+    milestone = models.ForeignKey("Milestone", null=True, blank=True, on_delete=models.SET_NULL, help_text="Association with a Workflow Level 1 Milestone")
+    # date_of_request = models.DateTimeField("Date of Request", blank=True, null=True)
```
Comment (jefmoura) [inline workflow/models.py]: Remove this field instead of commenting out.

File: indicators/models.py [hunk 2]
```diff
@@ -454,24 +454,24 @@ def get_queryset(self):
...
-    achieved = models.DecimalField("Achieved", max_digits=20, decimal_places=2, default=Decimal('0.00'))
-    disaggregation_value = models.ManyToManyField(DisaggregationValue, blank=True)
-    description = models.TextField("Remarks/comments", blank=True, null=True)
-    indicator = models.ForeignKey(Indicator)
-    workflowlevel2 = models.ForeignKey(WorkflowLevel2, blank=True, null=True, related_name="i_workflowlevel2", verbose_name="Project Initiation")
-    workflowlevel1 = models.ForeignKey(WorkflowLevel1, blank=True, null=True, related_name="i_workflowlevel1")
-    date_collected = models.DateTimeField(null=True, blank=True)
+    achieved = models.DecimalField("Achieved", max_digits=20, decimal_places=2, default=Decimal('0.00'), help_text="Actual or total for this record")
+    disaggregation_value = models.ManyToManyField(DisaggregationValue, blank=True, help_text="Values for each disaggregated field")
+    description = models.TextField("Remarks/comments", blank=True, null=True, help_text="HOw was this data collected")
```
Comment (jefmoura) [inline indicators/models.py]: `O` is in capital letter

File: indicators/models.py [hunk 3]
```diff
@@ -454,24 +454,24 @@ def get_queryset(self):
 
 
 class CollectedData(models.Model):
-    data_uuid = models.CharField(max_length=255,verbose_name='Data UUID', default=uuid.uuid4, unique=True, blank=True)
-    periodic_target = models.ForeignKey(PeriodicTarget, null=True, blank=True)
+    data_uuid = models.CharField(max_length=255,verbose_name='Data UUID', default=uuid.uuid4, unique=True, blank=True, help_text="")
+    periodic_target = models.ForeignKey(PeriodicTarget, null=True, blank=True, help_text="Releate this collection to a periodic target")
```
Comment (jefmoura) [inline indicators/models.py]: Type `Releate`

File: indicators/models.py [hunk 4]
```diff
@@ -322,43 +322,43 @@ def get_queryset(self):
...
+    method_of_analysis = models.CharField(max_length=255, null=True, blank=True, verbose_name="Method of Analysis", help_text="Method of Analysis")
+    information_use = models.CharField(max_length=255, null=True, blank=True, verbose_name="Information Use", help_text="Information Use")
+    reporting_frequency = models.ForeignKey(Frequency, null=True, blank=True, verbose_name="Frequency of Reporting", help_text="Frequency of Reporting")
+    quality_assurance = models.TextField(max_length=500, null=True, blank=True, verbose_name="Quality Assurance Measures", help_text="Quality Assurance Measures")
+    data_issues = models.TextField(max_length=500, null=True, blank=True, verbose_name="Data Issues", help_text="Problems with the data quality if any")
+    indicator_changes = models.TextField(max_length=500, null=True, blank=True, verbose_name="Changes to Indicator", help_text="How did the indicator change over time if at all")
+    comments = models.TextField(max_length=255, null=True, blank=True, help_text="")
+    workflowlevel1 = models.ManyToManyField(WorkflowLevel1, help_text="Related workflowlevel1 or project/prgoram etc.")
```
Comment (jefmoura) [inline indicators/models.py]: Typo `prgoram`

File: indicators/models.py [hunk 5]
```diff
@@ -322,43 +322,43 @@ def get_queryset(self):
 
 class Indicator(models.Model):
     indicator_uuid = models.CharField(max_length=255,verbose_name='Indicator UUID', default=uuid.uuid4, unique=True, blank=True)
-    indicator_type = models.ManyToManyField(IndicatorType, blank=True)
-    level = models.ForeignKey(Level, null=True, blank=True, on_delete=models.SET_NULL)
-    objectives = models.ManyToManyField(Objective, blank=True,verbose_name="Objective", related_name="obj_indicator")
-    strategic_objectives = models.ManyToManyField(StrategicObjective, verbose_name="Country Strategic Objective", blank=True, related_name="strat_indicator")
+    indicator_type = models.ManyToManyField(IndicatorType, blank=True, help_text="If indicator was pulled from aservie select one here")
```
Comment (jefmoura) [inline indicators/models.py]: Typo `aservie`

File: workflow/models.py [hunk 6]
```diff
@@ -1097,54 +1097,51 @@ class WorkflowLevel2(models.Model):
...
+    sector = models.ForeignKey("Sector", verbose_name="Sector", blank=True, null=True, related_name="workflow2_sector", on_delete=models.SET_NULL, help_text="Primary Sector or type of work")
+    sub_sector = models.ManyToManyField("Sector", verbose_name="Sub-Sector", blank=True, related_name="workflowlevel2_sub_sector", help_text="Secondary sector or type of work")
+    description = models.TextField("Description", blank=True, null=True, help_text="Description of the overall effort")
+    site = models.ManyToManyField(SiteProfile, blank=True, help_text="Geographic sites or locations")
+    short_name = models.CharField("Code", max_length=20, blank=True, null=True , help_text="Shortened name autogenerated")
+    office = models.ForeignKey(Office, verbose_name="Office", null=True, blank=True, on_delete=models.SET_NULL, help_text="Primary office for effort")
+    staff_responsible = models.ForeignKey(TolaUser, on_delete=models.SET_NULL, blank=True, null=True, help_text="Responsible party")
+    # partners = models.ForeignKey(Partner, blank=True, null=True, verbose_name="Partners", on_delete=models.SET_NULL)
```
Comment (jefmoura) [inline workflow/models.py]: Remove this field as well

File: workflow/models.py [hunk 7]
```diff
@@ -1097,54 +1097,51 @@ class WorkflowLevel2(models.Model):
...
-    agency_cost = models.DecimalField("Actual Cost for Organization", decimal_places=2, max_digits=12, help_text="In USD", default=Decimal("0.00"), blank=True)
-    community_handover = models.BooleanField("CommunityHandover/Sustainability Maintenance Plan", help_text='Check box if it was completed', default=False)
-    capacity_built = models.TextField("Describe how sustainability was ensured for this project?", max_length=755, blank=True, null=True)
-    quality_assured = models.TextField("How was quality assured for this project", max_length=755, blank=True, null=True)
-    issues_and_challenges = models.TextField("List any issues or challenges faced (include reasons for delays)", blank=True, null=True)
-    lessons_learned = models.TextField("Lessons learned", blank=True, null=True)
+    # agency_cost = models.DecimalField("Actual Cost for Organization", decimal_places=2, max_digits=12, help_text="In USD", default=Decimal("0.00"), blank=True)
+    # community_handover = models.BooleanField("CommunityHandover/Sustainability Maintenance Plan", help_text='Check box if it was completed', default=False)
```
Comment (jefmoura) [inline workflow/models.py]: Remove these fields too

File: workflow/models.py [hunk 8]
```diff
@@ -1097,54 +1097,51 @@ class WorkflowLevel2(models.Model):
...
+    capacity_built = models.TextField("Describe how sustainability was ensured for this project?", max_length=755, blank=True, null=True, help_text="Descriptive, did this help increases internal or external capacity")
+    quality_assured = models.TextField("How was quality assured for this project", max_length=755, blank=True, null=True, help_text="Descriptive, how was the overall quality assured for this effort")
+    issues_and_challenges = models.TextField("List any issues or challenges faced (include reasons for delays)", blank=True, null=True, help_text="Descriptive, what are some of the issues and challenges")
+    lessons_learned = models.TextField("Lessons learned", blank=True, null=True, help_text="Descriptive, when completed what lessons were learned")
     indicators = models.ManyToManyField("indicators.Indicator", blank=True)
     # sort array for activities related to a project
-    sort = models.IntegerField(default=0, blank=True)
+    # sort = models.IntegerField(default=0, blank=True)
```
Comment (jefmoura) [inline workflow/models.py]: Remove it

**Comment by jefmoura** [comment]
@glind Are you gonna add docstrings to describe the endpoints in the viewsets in another PR?

File: workflow/models.py (hunk 1 above)
Comment (glind) [reply workflow/models.py]: I don't want to remove the field until we know for sure it's not in use by the front end.  Commented code is intended for verification.

File: workflow/models.py (hunk 8 above)
Comment (glind) [reply workflow/models.py]: These are all fields that I need to confirm with Front-End they are not in use and comments make them stand out so I can share and confirm

**Pull request merged**